src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;1,700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="https://www.moyo.ua">
+        <img
+          src="./images/logo.png"
+          alt="Moyo"
+          class="logo">
+      </a>
+      <nav>
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="http://moyo.com.ua" class="header_link is-active">Apple</a>
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="http://moyo.com.ua" class="nav__link is-active">Apple</a>
```
Зверни увагу на назви класів